interface/patient_file/encounter/find_code_history.php

```diff
@@ -171,10 +171,10 @@ function setCodes() {
 </head>
 
 <body>
-    <div class="container-fluid w-100 position-fixed" style="margin-right: 10px;">
+    <div class="container-fluidposition-fixed">
         <div class="input-group bg-white">
             <div class="input-group-prepend">
-                <button class="btn btn-secondary" onclick='clearCodes(this)'><i class="fa fa-trash fa-1x"></i></button>
+                <button class="btn btn-danger" onclick='clearCodes(this)'><i class="fa fa-trash fa-1x"></i></button>
```

Comment:
Hi @bradymiller  and @tywrenn 

The #LINE186 contains `bg-warning` class, but it let the view looks like odd, should I remove it?

![image](https://user-images.githubusercontent.com/22230889/90094487-a26ef100-dd60-11ea-915c-997c5a537b2f.png)

Reply:
Seems useless so you can remove it

Reply:
@stu01509 
umm, that line doesn't do what you think! its for Help
![image](https://user-images.githubusercontent.com/1859985/90347760-432f0c00-e000-11ea-8578-e52fa218f3e5.png)

Also, if you don't select a procedure then this dialog will highlight all history for user. Still the dialog would not be used unless user has already selected a procedure. This is how it looks with procedure selected. So, changing L186 has nothing to do with highlights. You can move the bg class up to L-182 and change color if wanted as long as it contrast with dialog so in just doesn't blend into dialog background.

![image](https://user-images.githubusercontent.com/1859985/90347710-0236f780-e000-11ea-9df6-23fe79200d16.png)

so bottom line, please be careful making assumptions when removing anything and check JS to make sure something is not dynamically being used. Also, don't fool with the floating work bar. I want it to work the way it work now...